app/models/user.rb

```diff
@@ -364,8 +364,8 @@ def cached_followed_tag_names
   end
 
   # methods for Administrate field
-  def banned
-    has_role? :banned
+  def suspended
+    has_role? :suspended
```

Comment:
@msarit @jacobherrington  In general, this shouldn't be a problem because `has_role?` performs a database lookup and the data update script updates the role name there. BUT: the scripts don't run during deployment, but get scheduled to run in ~5 minutes. During this short window the role would still be called `:banned` but checks would be performed against `:suspended`. So we should indeed keep it like this for the initial deployment but can remove it after.